src/pages/research_profile/ResearchProfile.tsx

```diff
@@ -145,6 +161,38 @@ const ResearchProfile = () => {
           )}
         </>
       )}
+
+      <Divider />
+      <Typography id="project-label" variant="h2" sx={{ mt: '1rem' }}>
+        {t('my_page.my_profile.projects')} ({projectsQuery.data && projectsQuery.data.size})
+      </Typography>
+      {projectsQuery.isLoading ? (
+        <Box sx={{ display: 'flex' }}>
+          <CircularProgress sx={{ justifySelf: 'center' }} aria-labelledby="project-label" />
+        </Box>
```

Comment:
ser ikke ut som dette trengs, eller?
```suggestion
          <CircularProgress aria-labelledby="project-label" />
```